compiler/luci-interpreter/src/kernels/SplitV.cpp

```diff
@@ -43,6 +43,20 @@ void SplitV::configure()
   auto sizes_data = getTensorData<int32_t>(size_splits());
 
   assert(size_splits()->shape().num_dims() == 1);
+  // Check if last elem in size_splits is -1 and change it to right dim value
+  const auto num_dims_size_spits = size_splits()->shape().dim(0);
+  if (sizes_data[num_dims_size_spits - 1] == -1)
+  {
+    auto non_const_sizes_data = const_cast<int32_t *>(sizes_data);
+    int32_t sum = 0;
+    for (int32_t i = 0; i < num_dims_size_spits - 1; ++i)
+    {
+      assert(sizes_data[i] > 0);
+      sum += sizes_data[i];
+    }
+    non_const_sizes_data[num_dims_size_spits - 1] = input()->shape().dim(_axis_value) - sum;
```

Comment:
hmmm, what if this value will change in next inference?

Reply:
According tflite sources, -1 could be found on any position. I think it is better to process this in the loop below.

For example, like this:
``` c++
  auto output_shape = input()->shape();
  for (int32_t i = 0; i < num_split; ++i)
  {
    if (sizes_data[i] == -1)
    {
      output_shape.dim(_axis_value) = input()->shape().dim(_axis_value) - sum;
    }
    else
    {
      output_shape.dim(_axis_value) = sizes_data[i];
    }
    _outputs[i]->resize(output_shape);
  }
```